src/app/rootReducer.ts

```diff
@@ -1,10 +1,12 @@
-import {combineReducers} from '@reduxjs/toolkit'
-import {baseApi, jwtApi} from '@/shared/api'
+import {combineReducers} from '@reduxjs/toolkit';
+import {baseApi, jwtApi} from '@/shared/api';
 import {SessionSlice} from "@/entities/session/model/slice";
-import {registerFormSlice} from '@/features/authentication/registration/model/formState'
+import globalFilterReducer from '@/features/globalFilter/model/GlobalFilterSlice';
+import {registerFormSlice} from '@/features/authentication/registration/model/formState';
 
 export const rootReducer = combineReducers({
   session: SessionSlice.reducer,
+  globalFilter: globalFilterReducer,
```

Comment:
I would suggest to rename globalFilter to searchFilter because you have changed the name of that component

Reply:
globalFilter is a slice for all filters including filters from drop-down menu. i have no maquette for drop-down, thats why i cant add more filters in this slice yet

Reply:
but still this filter is for search, why don't call it search? global filter doesn't describe that this filter is used for searching an event